client/src/components/composables/media-player.ts

```diff
@@ -176,3 +176,8 @@ export function usePlaybackRate() {
 		availablePlaybackRates,
 	};
 }
+
+export enum MediaPlayerError {
+	none = 0,
+	network = 1,
```

Comment:
If there is no media player error, then it should that should be represented with `null` or `undefined`, not as a part of the error.

In other words, if its an error, its some value. If there's no error, its null/undef.